lua/graph.lua

```diff
@@ -931,7 +931,7 @@ function filter_tags_generic(kv)
     forward = highway[kv["construction"]]
   end
   local ferry = kv["route"] == "ferry"
-  local rail = kv["route"] == "shuttle_train"
+  local rail = kv["route"] == "shuttle_train" or kv["railway"] == "rail"
```

Comment:
Needed for [LeShuttle](https://www.openstreetmap.org/way/143253058) and a few other "rail shuttles" in Switzerland like this one - https://www.openstreetmap.org/way/643895309

Reply:
oh god, it is surprisingly hard to do it properly - treating rails as rails makes them "enable all by default", which is right for `shuttle_train`, but not right for regular trains (`kv["railway"] == "rail"`).